tests/functional/SqlSyncTest.php

```diff
@@ -25,58 +25,68 @@ public function testSimulatedSqlSync()
         $output = $this->getSimplifiedErrorOutput();
         $this->assertContains("[notice] Simulating: ssh -o PasswordAuthentication=whatever www-admin@server.isp.simulated '/path/to/drush sql-dump --no-interaction --strict=0 --gzip --result-file=auto --format=json --uri=remote --root=__DIR__/sut", $output);
         $this->assertContains("[notice] Simulating: __DIR__/drush core-rsync @synctest.remote:/simulated/path/to/dump.tgz @synctest.local:__SANDBOX__/tmp/dump.tgz --yes --uri=local --root=__DIR__/sut -- --remove-source-files", $output);
         $this->assertContains("[notice] Simulating: __DIR__/drush sql-query --no-interaction --strict=0 --file=__SANDBOX__/tmp/dump.tgz --file-delete --uri=local --root=__DIR__/sut", $output);
 
         // Test simulated simple sql:sync local-to-remote
-        $this->drush('sql:sync', ['@synctest.local', '@synctest.remote'], $options, '@synctest.local');
+        $this->drush('sql:sync', [
+                '@synctest.local',
+                '@synctest.remote',
+        ], $options, '@synctest.local');
         $output = $this->getSimplifiedErrorOutput();
         $this->assertContains("[notice] Simulating: __DIR__/drush sql-dump --no-interaction --strict=0 --gzip --result-file=auto --format=json --uri=local --root=__DIR__/sut", $output);
         $this->assertContains("[notice] Simulating: __DIR__/drush core-rsync @synctest.local:/simulated/path/to/dump.tgz @synctest.remote:/tmp/dump.tgz --yes --uri=local --root=__DIR__/sut -- --remove-source-files", $output);
         $this->assertContains("[notice] Simulating: ssh -o PasswordAuthentication=whatever www-admin@server.isp.simulated '/path/to/drush sql-query --no-interaction --strict=0 --file=/tmp/dump.tgz --file-delete --uri=remote --root=__DIR__/sut'", $output);
 
         // Test simulated remote invoke with a remote runner.
-        $this->drush('sql:sync', ['@synctest.remote', '@synctest.local'], $options, 'user@server/path/to/drupal#sitename');
+        $this->drush('sql:sync', [
+                '@synctest.remote',
+                '@synctest.local',
+        ], $options, 'user@server/path/to/drupal#sitename');
         $output = $this->getSimplifiedErrorOutput();
         $this->assertContains("[notice] Simulating: ssh -o PasswordAuthentication=no user@server 'drush --no-interaction sql:sync @synctest.remote @synctest.local --uri=sitename --root=/path/to/drupal'", $output);
     }
 
     /**
      * Covers the following responsibilities.
      *   - A user created on the source site is copied to the destination site.
-     *   - The email address of the copied user is sanitized on the destination site.
+     *   - The email address of the copied user is sanitized on the destination
+     * site.
      *
      * General handling of site aliases will be in sitealiasTest.php.
      */
     public function testLocalSqlSync()
     {
-        if ($this->dbDriver() == 'sqlite') {
+        if ($this->dbDriver()=='sqlite') {
```

Comment:
This is not how we format code. Dont think this is required by PSR2

Reply:
This thing is strange, I didn't change anything here.